doc/source/whatsnew/v0.23.0.txt

```diff
@@ -690,6 +690,7 @@ I/O
 ^^^
 
 - :func:`read_html` now rewinds seekable IO objects after parse failure, before attempting to parse with a new parser. If a parser errors and the object is non-seekable, an informative error is raised suggesting the use of a different parser (:issue:`17975`)
+- :func:`read_html` now has an option to add an id to the leading `<table>` tag (:issue:`8496`)
```

Comment:
`to_html`, not read. Also it's a method, so

    :meth:`DataFrame.to_html` now has ...

Reply:
Thanks! Changed.